resources/prefs.json

```diff
@@ -56,6 +56,7 @@
   "layout.viewport.enabled": false,
   "layout.writing-mode.enabled": false,
   "network.mime.sniff": false,
+  "session_history.max_length":20,
```

Comment:
Let's use `session-history.max-length` to be consistent with other preference names. Also, add a space after `:`.